src/blocks/venue/style.scss

```diff
@@ -1 +1,6 @@
 @import "../venue-information/style";
+
+.wp-block-gatherpress-venue > iframe {
+    height: 400px;
+	width: 100%;
```

Comment:
Fix indenting. Good idea to turn white space visibility on in your IDE so it's easy to spot.